master/internal/api_user.go

```diff
@@ -88,6 +95,10 @@ func getUser(d *db.PgDB, userID model.UserID) (*userv1.User, error) {
 	return toProtoUserFromFullUser(*user), nil
 }
 
+func isSSOUser(_ model.User) bool {
```

Comment:
I'd recommend putting it into a separate file within `internal/user/` package. otherwise any big change on `api_user.go` will tend to conflict.